scripts/apps/search/directives/RawSearch.js

```diff
@@ -1,12 +1,22 @@
-RawSearch.$inject = ['asset', '$location'];
+/**
+ * @memberof superdesk.apps.search
```

Comment:
Thanks Gabriel, I have made the changes! 
(When looking for an examples of documenting directives I must have chosen badly!)